code-name-taran/src/com/spaghettisoft/component/scrabble/TurnMakeWord.java

```diff
@@ -0,0 +1,185 @@
+package com.spaghettisoft.component.scrabble;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class TurnMakeWord {
+
+	Scanner scanner;
+	String word;
+	Stave stave;
+
+	TurnMakeWord(Scanner scanner) {

```

Comment:
Just an observation - This constructor is with package level access. Are you sure that this is what you have intended to do :P